docs/UserGuide.md

```diff
@@ -330,8 +330,8 @@ Format: `edit -e INDEX [d/DESCRIPTION] [at/DATE_TIME]
 * The index must be a positive integer 1, 2, 3, ...
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* `[ap/CONTACT_INDEX_LIST]` adds contacts with the specified indexes to the event.
-* `[rp/ATTENDEE_INDEX_LIST]` removes contacts with the specified indexes from the event.
+* `[ap/CONTACT_INDEX_LIST]` add contacts with the specified indexes to the event.
+* `[rp/ATTENDEE_INDEX_LIST]` remove contacts with the specified indexes from the event.
```

Comment:
'Indices' instead of 'indexes'

Reply:
Fixed

Reply:
Grammatically both works, Im gonna standardise it to indices.